descriptor/extensions.go

```diff
@@ -137,6 +137,7 @@ func getLoaderOption(method *Method, value string, many bool) *Loader {
 		Many:              many,
 		RequestFieldPath:  strings.Split(parts[1], "."),
 		ResponseFieldPath: strings.Split(parts[2], "."),
+		ObjectKeyPath:     strings.Split(parts[3], "."),
```

Comment:
When we deploy this, we should make sure that the custard thing won't break the world, we'll do it really quickly in succession, but ideally if there are protos that don't have the 3rd part, it won't fall over